src/package-vsix.js

```diff
@@ -17,26 +17,27 @@ const repository = {
     "url": "https://github.com/theia-ide/vscode-builtin-extensions"
 };
 
-// bump to publish
-let version = '1.39.1-prel';
+const vscodePck = JSON.parse(fs.readFileSync(vscode('package.json'), 'utf-8'));
+let version = vscodePck.version || '0.0.1';
 
 (async () => {
-
     const bin = await run('yarn', ['bin'], root());
     const vsce = bin.trim() + '/vsce';
 
+    // use VS Code version and SHA when publishing next
     if (tag === 'next') {
         const shortRevision = (await run('git', ['rev-parse', '--short', 'HEAD'], vscode())).trim();
-        const [, minor] = version.split('.');
-        version = `0.${Number(minor) + 1}.0-next.${shortRevision}`;
+        const [major, minor , ] = version.split('.');
+        version = `${major}.${Number(minor) + 1}.0-next.${shortRevision}`;
```

Comment:
@spoenemann do you think this way of versioning a `next` or `insider` will work ok when publishing on open-vsx.org? e.g. for the current vscode `1.44.2` it will give a version like `1.45.0-next.ff91584411`

Reply:
A `next` version should also set `preview` to `true` in its package.json, but then I think it's fine.

Currently the Open VSX website displays the latest published version, including preview versions. We need to think about whether we want to go in the direction of npmjs.com: display the `latest` version (excluding next/preview) as default, but allow to choose any other version in the UI.

Reply:
> We need to think about whether we want to go in the direction of npmjs.com: display the latest version (excluding next/preview) as default, but allow to choose any other version in the UI.

Yes, that's my main worry - whether we should do that and how. 

For now I have added the `preview=true` flag when packaging "next" versions of builtins, and I have made "next" the default when one calls "yarn".